Eatery Blue/UI/MenusScreen/MenuDayPickerTableViewCell.swift

```diff
@@ -78,12 +83,12 @@ class MenuDayPickerTableViewCell: UITableViewCell {
         contentView.addSubview(dayPickerView)
     }
     
-    private func setState() {
-        days = []
-        for i in 0...6 {
-            days.append(Day().advanced(by: i))
-        }
-    }
+//    private func setState() {
```

Comment:
should we get rid of this because its unused?

Reply:
Yes. @cassidyxu Remove this since it is no longer being used.